unlock-app/src/components/interface/icons/Eth.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 const Eth = props => (
-  <svg viewBox="0 0 7 11" width="1em" height="1em" {...props}>
```

Comment:
nope... these are automatically generated so we cannot change the values in there (I need to find a way to indicate that).
If you need the icons to be larger just change the `font-size` of the element in which the icons are displayed.

Reply:
Ah got it. OK!